spk/mosh/Makefile

```diff
@@ -12,7 +12,7 @@ DESCRIPTION = mosh, the mobile shell is a remote terminal application that allow
 
 DISPLAY_NAME = mosh
 STARTABLE = no
-CHANGELOG = "1. Add libutempter to show detached sessions.<br/>2. Update openssl to v1.1.1l.<br/>3. Add Perl dependency for advanced customization and scripting."
+CHANGELOG = "1. Update Mosh to version 1.4.0."
```

Comment:
probably we should add the info, that openssl is updated to 1.1.1t since the last release.

Reply:
Done.